src/package-manager.js

```diff
@@ -869,4 +887,54 @@ module.exports = class PackageManager {
       normalizePackageData(metadata)
     }
   }
+
+  async fetchBlacklist () {
+    try {
+      const response = await window.fetch(BLACKLIST_API_URL)
+      const blacklist = await response.json()
+      return new Set(blacklist)
+    } catch (e) {
+      return new Set()
```

Comment:
This should log the error somewhere so we at least know when it's failing. Maybe `console.warn`?